manifests/kots-config.yaml

```diff
@@ -923,21 +871,18 @@ spec:
           title: Advanced Platform Tuning
           type: bool
           default: "0"
-          help_text: |-
-            ⚠️ Check only for tuning your CARTO Self-Hosted environment with advanced parameters
+
 
     # Advanced Tuning
     - name: platformAdvancedTuning
       title: Advanced Tuning
       description: |-
-        Here you can config advanced options for tuning your CARTO deployment.
-        If you need further info, please check [docs.carto.com](https://docs.carto.com).
+        Advanced Configuration Section where you can enter plaintext configuration settings, provided by the CARTO support team, to tailor your installation to specific requirements.
+        
       when: repl{{ ConfigOptionEquals "enablePlatformAdvancedTuning" "1" }}
       items:
         ## Tunning
         - name: platformAdvancedTuningValues
-          help_text: |-
```

Comment:
I think that we should specify something like this because the client could totally break the installation putting things here and... we are not responsible of it :see_no_evil: